Modules/Quotas/src/Quotas.Domain/Aggregates/Relationships/RelationshipTemplate.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace Backbone.Modules.Quotas.Domain.Aggregates.Relationships;
 public class RelationshipTemplate : ICreatedAt
 {
-    public string CreatedBy { get; set; }
+    public string? CreatedBy { get; set; }
```

Comment:
Not sure but I think this should not be nullable either.

Reply:
👍